.gitattributes

```diff
@@ -4,6 +4,7 @@
 
 **/zz_generated.*.go linguist-generated=true
 /pkg/client/** linguist-generated=true
+/contrib/*/pkg/client/** linguist-generated=true
```

Comment:
what's this?

Reply:
Please ignore, merging different branches issue. Just resolved now.